File: samples/sample.cs
using System;
using System.Collections.Generic;

// Class
public class FooBar
{
    public string Foo { get; set; }
    public int Bar { get; set; }

    public FooBar(string foo, int bar)
    {
        this.Foo = foo;
        this.Bar = bar;
    }

    public void Print()
    {
        Console.WriteLine($"Foo: {this.Foo}, Bar: {this.Bar}");
    }
}

public class Program
{
    public static void Main(string[] args)
    {
        // Using the class
        var foobar = new FooBar("Lorem Ipsum", 123);
        foobar.Print();

        // Using a list
        var baz = new List<int> { 1, 2, 3, 4, 5 };
        foreach (var i in baz)
        {
            Console.WriteLine(i);
        }

        // Using a dictionary
        var qux = new Dictionary<string, string>
        {
            { "foo", "bar" },
            { "baz", "qux" }
        };

        foreach (var kvp in qux)
        {
            Console.WriteLine($"Key: {kvp.Key}, Value: {kvp.Value}");
        }

        // Using control structures
        if (qux.ContainsKey("foo"))
        {
            Console.WriteLine($"Found foo: {qux["foo"]}");
        }
        else
        {
            Console.WriteLine("Did not find foo");
        }

        // Using a switch statement
        switch (foobar.Bar)
        {
            case 123:
                Console.WriteLine("Bar is 123");
                break;
            default:
                Console.WriteLine("Bar is not 123");
                break;
        }
    }
}

File: samples/sample.php
<?php

// Class
class FooBar {
    public $foo;
    public $bar;

    public function __construct($foo, $bar) {
        $this->foo = $foo;
        $this->bar = $bar;
    }

    public function print() {
        echo "Foo: " . $this->foo . ", Bar: " . $this->bar . "\n";
    }
}

// Using the class
$foobar = new FooBar("Lorem Ipsum", 123);
$foobar->print();

// Array
$baz = array(1, 2, 3, 4, 5);
foreach ($baz as $i) {
    echo $i . "\n";
}

// Associative array
$qux = array("foo" => "bar", "baz" => "qux");
foreach ($qux as $key => $val) {
    echo "Key: " . $key . ", Value: " . $val . "\n";
}

// Conditional
if (array_key_exists("foo", $qux)) {
    echo "Found foo: " . $qux["foo"] . "\n";
} else {
    echo "Did not find foo\n";
}

// Switch statement
switch ($foobar->bar) {
    case 123:
        echo "Bar is 123\n";
        break;
    default:
        echo "Bar is not 123\n";
        break;
}
?>
